backend/orcid.py

```diff
@@ -146,6 +149,35 @@ def warn_user_of_ignored_papers(self, ignored_papers):
                                     notification,
                                     'backend_orcid'
                                     )
+            
+    def link_existing_papers(self, researcher):
+        """
+        Search for papers which bear the ORCID id of the researcher,
+        but which are not linked to the researcher itself, and updates them
+        to link to the researcher.
+        """
+        queryset = SearchQuerySet().models(Paper).filter(orcids=researcher.orcid).load_all()
+        papers_to_update = []
+        for search_result in queryset:
+            paper = search_result.object
+            if researcher.id not in paper.researcher_ids:
+                new_authors = paper.authors
+                for author in new_authors:
+                    if author.orcid == researcher.orcid:
+                        author.researcher_id = researcher.id
+                paper.authors_list = [author.serialize() for author in new_authors]
+                papers_to_update.append(paper)
+                paper.update_index()
+        
+        if papers_to_update:
+            if django.VERSION[0] > 2 or (django.VERSION[0] == 2 and django.VERSION[1] >= 2):
```

Comment:
Is there a main reason to support multiple Django versions?

Reply:
Yeah let's just require 2.2, this `bulk_update` thing is going to be useful at many other places I think